stdlib/array.gr

```diff
@@ -296,9 +312,7 @@ export let forEachi = (fn, array) => {
  */
 export let map = (fn, array) => {
   let length = length(array)
-  init(length, (i) => {
-    fn(array[i])
-  })
+  init(length, i => {fn(array[i])})
```

Comment:
@marcusroberts this should either:
1. Remove the brackets
2. Always be rendered as a block because it has brackets

Reply:
Going for option 2!